verification/src/transaction_verifier.rs

```diff
@@ -96,10 +99,11 @@ where
                 epoch_number_with_fraction,
                 parent_hash,
             ),
+            fee_calculator: FeeCalculator::new(rtx, &consensus, &chain_store),
         }
     }
 
-    pub fn verify(&self, max_cycles: Cycle) -> Result<Cycle, Error> {
+    pub fn verify(&self, max_cycles: Cycle) -> Result<CacheEntry, Error> {
```

Comment:
https://github.com/nervosnetwork/ckb/blob/develop/chain/src/chain.rs#L631-L636
develop branch move out calculate fee from `Verifier`,
you need to merge this.